src/layouts/components/Editor/extensions/Image/Image.ts

```diff
@@ -85,6 +86,26 @@ export const IsomerImage = Image.extend({
         return true
       },
 
+      setImageMeta: (options) => ({ tr, dispatch, editor }) => {
```

Comment:
this method has the same signature as `setImage` - we probably _don't_ want this as this method should only allow devs to set certain meta level attributes post image creation (so things like `src` cannot be set). 

given this, can we tighten options here? perhaps a quick fix would be to just use the type `Pick<IsomerImageOptions, 'alt' | 'href'>`. this makes it obvious that this method cannot be used to set other attributes (atm it can) and make it distinct from `setImage`

Reply:
@seaerchin Sure, this makes sense and agree that it's better for maintainability. Other than just `alt` and `href`, I want to allow this method to set more attributes so it can fulfil future use cases. How about the following list:
```
  alt?: string
  title?: string
  width?: string
  height?: string
  href?: string
```
`style` shouldn't be allowed otherwise it's risky cause the string can have wrong value. They should only be able to set style by using `setImageStyle`. Similarly, `src` is not allowed. 
wdyt?

Reply:
we can leave it as just `href/alt` first and add in when there's a need. we should avoid having [god methods](https://wiki.c2.com/?GodMethod) that do everything and add on a as-required basis.

Reply:
@seaerchin Sure, done it in ff27724e206dd2a25b3d3eed7088be9c6e496bc8